### PR TITLE
[FOGL-613] storage service discovery by name; start core then start start storage

### PR DESCRIPTION
**Pull request by praveen-garg**
enjoy with payload builder and storage client; allow few seconds to storage to register itself

### CODE REVIEW

**Comment by praveen-garg** [comment]
#222

**Comment by praveen-garg** [comment]
comment foglamp.core.server_daemon.Daemon#_configure_logging to keep it working in daemon. FOGL-583 fix coming just after its merge.

File: src/python/foglamp/storage/storage.py [hunk 1]
```diff
@@ -129,7 +122,35 @@ def check_shutdown(self):
         conn.close()
         return json.loads(res)
 
+    def get_storage_service(self):
+        """ get Storage service """
+
+        # TODO: URL to service registry api?
+        conn = http.client.HTTPConnection("localhost:8082")
```
Comment (MarkRiddoch) [inline src/python/foglamp/storage/storage.py]: This will come from the command line arguments eventually

Comment (praveen-garg) [reply src/python/foglamp/storage/storage.py]: Yupe with FOGL-517 and start up fix. Hence a TODO.

**Pull request merged**